### PR TITLE
refinem (0.0.24) build 3

### PR DESCRIPTION
**Pull request by HadrienG**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] ~~This PR adds a new recipe.~~
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] ~~This PR does something else (explain below).~~

### CODE REVIEW

**Comment by HadrienG** [comment]
Hi @bioconda/core ! This is my first recipe and I'm running into some issues.

All my PRs and commits on this recipe end up with:

```bash
16:44:27 BIOCONDA INFO Nothing to be done.
```

I cannot figure why my recipe does not build. Any ideas?
First PR #12628

Thank you in advance!

File: recipes/refinem/meta.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ source:
   sha256: "{{ sha256 }}"
 
 build:
-  number: 1
+  number: 3
```
Comment (druvus) [inline recipes/refinem/meta.yml]: Build number should be 2

File: recipes/refinem/meta.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ source:
...
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
   skip: True  # [not py27]
```
Comment (druvus) [inline recipes/refinem/meta.yml]: Remove this line. Python is restricted below.

File: recipes/refinem/meta.yml [hunk 3]
```diff
@@ -23,15 +23,15 @@ requirements:
     - mpld3 >=0.2
     - numpy >=1.9.0
     - pip
-    - python
+    - python < 3
```
Comment (druvus) [inline recipes/refinem/meta.yml]: should be `python <3`, i. e no white space.

**Comment by druvus** [comment]
I think the issue was that your file had extension `yml` instead of `yaml`.

**Comment by druvus** [comment]
@HadrienG the dependency `weightedstats` is not yet available in bioconda/conda-forge. Could you please add it to conda-forge as it is a generic package.

**Comment by HadrienG** [comment]
@druvus Thanks a lot for the review! Will fix `weightedstats` in the morning.

**Comment by bgruening** [comment]
Closing in favor of cf package.

**Comment by apeltzer** [comment]
I think this was a mistake: `refinem` isn't available in conda-forge or bioconda. We just need to wait until weightedstats is taken into conda-forge (after @HadrienG 's changes) and could then proceed / retrigger the tests on this recipe.

**Comment by HadrienG** [comment]
`weightedstats` has been merged in `conda-forge`. Can someone retrigger the build?

**Comment by druvus** [comment]
@HadrienG restarted the test

**Pull request merged**